docs/rules.md

```diff
@@ -472,13 +472,6 @@ Location header must be supported for all async operations that return 202.
 
 Please refer to [lro-location-header.md](./lro-location-header.md) for details.
 
-### PostResponseCodes
-
```

Comment:
Why are we deleting this ?

Reply:
These are rush regen changes, I haven't manually deleted anything in this file.

Reply:
Okay! I see that https://github.com/Azure/azure-openapi-validator/blob/main/docs/rules.md#postresponsecodes-1 is the right description 
and the one thats getting removed is old